api/src/auth/auth.service.ts

```
@@ -75,12 +75,16 @@ export class AuthService {
       await this.usersService.updateConfirmEmailInfo(user.email, confirmEmailInfo);
     }
 
-    await this.mailService.sendMail({
+    const result = await this.mailService.sendMail({
       subject: this.i18n.translate(locale, 'confirmationEmail.body'),
       text:
         this.i18n.translate(locale, 'confirmationEmail.body') + '\n\n' + `Code : ${confirmEmailInfo.confirmEmailCode}`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)